src/backend/aspen/app/views/tests/test_views.py

```diff
@@ -15,6 +16,41 @@ def test_usergroup_view(session, app, client):
     assert expected == json.loads(res.get_data(as_text=True))
 
 
+def test_usergroup_view_put_pass(session, app, client):
+    group = group_factory()
+    user = user_factory(group, agreed_to_tos=False)
+    session.add(group)
+    session.commit()
+    with client.session_transaction() as sess:
+        sess["profile"] = {"name": user.name, "user_id": user.auth0_user_id}
+
+    data = {"agreed_to_tos": True}
+    res = client.put("/api/usergroup", json=json.dumps(data))
+
+    # start a new transaction
+    session.close()
+    session.begin()
```

Comment:
I suspect `session.commit()` is enough to force a new transaction to start.

Reply:
interesting, i guess it would